main.go

```diff
@@ -37,6 +37,14 @@ func main() {
 	if len(os.Args) > 1 {
 		emailToAdd := os.Args[1]
 		if validateEmail(emailToAdd) {
+
+			emailExist := doEmailExists(emailList, emailToAdd)
+			if emailExist {
+				fmt.Println("Email is already in list")
+				fmt.Printf("List: %v", emailList)
+				return
```

Comment:
Nice one! I like how you work with functions to solve the issue - good job bro
# 🎨